src/main/markdown/doc/latest/DevGuideCompileReport.md

```diff
@@ -1,3 +1,6 @@
+Compile Reports
```

Comment:
I agree with you; I went for consistency with the rest of the site, although Compilation reports sounds better.


> On 25 feb. 2015, at 20:01, Julien Dramaix <notifications@github.com> wrote:
> 
> In src/main/markdown/doc/latest/DevGuideCompileReport.md:
> 
> > @@ -1,3 +1,6 @@
> > +Compile Reports
> Compilation reports or compiler reports don't sound better ?
> 
> —
> Reply to this email directly or view it on GitHub.
>